### PR TITLE
Complete user stories for favorites

### PR DESCRIPTION
**Pull request by priyapower**
Author: Priya
Contributions: Priya & Roberto

- Users will be able to favorite a pet and keep track of pets they are interested in; Transferred consistent web page information or data into application.html.erb; Transitioned to Bootstrap for CSS styling; Corrected web tab-naming for Adopt-Paired (eventually update this to "Adopt Don't Shop")
- Improvements: Refactor all "business logic" from views into controllers and poro models
- Edge Cases: This is taken care of from user stories, as far as we can see
- View: Create top nav bar across any size site && bottom links bar across any size site; Investigate overriding application.html.erb for welcome/root page; Adding consistent imaging/logoing to every webpage (through application.html.erb)

### CODE REVIEW

**Comment by robertorodriguez12** [comment]
Reviewed by: Roberto

File: app/controllers/favorites_controller.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 class FavoritesController < ApplicationController
   include ActionView::Helpers::TextHelper
 
+  def index
+    favorites
```
Comment (robertorodriguez12) [inline app/controllers/favorites_controller.rb:5]: i like that we can just call. favorites now. Good Job!!!

File: app/controllers/favorites_controller.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+  end
+
   def update
     pet = Pet.find(params[:pet_id])
     # @favorite = Favorite.new(session[:favorite])
```
Comment (robertorodriguez12) [inline app/controllers/favorites_controller.rb]: looks like we can remove this now. what do you think?

File: app/controllers/favorites_controller.rb [hunk 3]
```diff
@@ -10,4 +14,18 @@ def update
...
     redirect_to "/pets/#{pet.id}"
   end
+
+  def destroy
+    pet = Pet.find(params[:pet_id])
+    favorites.remove_pet(pet.id)
+    session[:favorite] = favorites.contents
+    quantity = favorites.count
+    flash[:remove_favorite] = "This pet was removed from My Favorites. You now have #{pluralize(quantity, "favorite")}"
+    redirect_to request.referer
```
Comment (robertorodriguez12) [inline app/controllers/favorites_controller.rb:23]: This one is awesome!!!! Good Job!!! Thank you for explaining this to me.

**Review by robertorodriguez12 — CHANGES_REQUESTED** [review]
Let's remove unnecessary comments.

File: app/controllers/favorites_controller.rb (hunk 1 above)
Comment (priyapower) [reply app/controllers/favorites_controller.rb:5]: :)

File: app/controllers/favorites_controller.rb (hunk 2 above)
Comment (priyapower) [reply app/controllers/favorites_controller.rb]: Absolutely; all comments are now removed; please confirm on your next code-review

File: app/controllers/favorites_controller.rb (hunk 3 above)
Comment (priyapower) [reply app/controllers/favorites_controller.rb:23]: For future research: https://api.rubyonrails.org/classes/ActionController/Redirecting.html

File: app/controllers/favorites_controller.rb (hunk 2 above)
Comment (robertorodriguez12) [reply app/controllers/favorites_controller.rb]: Confirmed!

**Review by robertorodriguez12 — APPROVED** [review]

**Pull request merged**